Remove now unneeded dependencies (webpack, babel, terser)

peaBerberian opened the pull request:
This commit removes dependencies that should now be unneeded.
    
I had to update the standalone demo's script to remove its dependency on  webpack by copy-pasting most of its bundling code from the full demo's script.
    
As we never really had a use for the standalone demo, I'll also propose in the future to just remove it, but I prefer to do that as a second step.

Florent-Bouisset commented on package.json:
```diff
@@ -226,13 +221,10 @@
     "semver": "7.5.4",
     "sinon": "17.0.1",
-    "terser-webpack-plugin": "5.3.9",
     "ts-loader": "9.5.1",
```
ts-loader is the typescript loader for webpack and can also be removed

peaBerberian replied:
Done

peaBerberian committed:
Remove now unneeded dependencies (webpack, babel, terser)

This commit removes dependencies that should now be unneeded.

I had to update the standalone demo's script to remove its dependency on
webpack by copy-pasting most of its bundling code from the full demo's
script.

As we never really had a use for the standalone demo, I'll also propose
in the future to just remove it, but I prefer to do that in a second
step.
